app.rb

```diff
@@ -46,6 +46,34 @@
 Cuba.define do
   persist_session!
 
+  # We set the 'guests' layout as master template (file: /views/layout.guests.mote)
+  settings[:mote][:layout] = "layout.guests"
+
+  # Petition page
+  on "petition/:slug" do |slug|
+    render("guests/petition", title: "SumOfUs")
+  end
+
+  # Donation page
+  on "donation/:slug" do |slug|
+    render("guests/donation", title: "SumOfUs")
+  end
+
+  # Donate page (for a general donation not related to any specific campaign)
+  on "donate" do
+    render("guests/donate", title: "SumOfUs")
+  end
+
+  # Campaigns page
+  on "campaigns" do
```

Comment:
I'm not sure we need this route at the moment - we don't have a concept of campaigns in the system right now, and right now, nobody actually views a whole campaign, because a campaign is almost always a collection of petitions that are related because they're the same petition in different languages.